package.json

```diff
@@ -20,43 +23,48 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.5.0",
+    "broccoli-asset-rev": "^2.4.2",
     "ember-blog-engine": "git://github.com/srowhani/ember-blog-engine.git#fc083eb7da328e1cbb338225c3cb628d684dbc3c",
-    "ember-browserify": "1.1.11",
-    "ember-cli": "~2.5.0",
+    "ember-browserify": "~1.1.0",
+    "ember-cli": "~2.8.0",
     "ember-cli-app-version": "^1.0.0",
-    "ember-cli-code-coverage": "0.3.4",
+    "ember-cli-code-coverage": "~0.3.0",
     "ember-cli-dependency-checker": "^1.2.0",
-    "ember-cli-deploy": "1.0.0-beta.1",
+    "ember-cli-deploy": "~0.6.4",
     "ember-cli-deploy-frost-pack": "git://github.com/ciena-frost/ember-cli-deploy-frost-pack.git#e98e53709a42604815348f10bc8823c3c425b68e",
     "ember-cli-htmlbars-inline-precompile": "^0.3.1",
     "ember-cli-inject-live-reload": "^1.4.0",
     "ember-cli-mocha": "^0.10.1",
     "ember-cli-notifications": "^4.0.9",
+    "ember-cli-test-loader": "^1.1.0",
     "ember-cli-uglify": "^1.2.0",
+    "ember-cli-visual-acceptance": "2.2.0",
     "ember-code-snippet": "1.4.0",
-    "ember-computed-decorators": "0.2.2",
-    "ember-concurrency": "0.7.10",
+    "ember-computed-decorators": "~0.2.0",
+    "ember-concurrency": "~0.7.0",
     "ember-disable-proxy-controllers": "^1.0.1",
     "ember-elsewhere": "0.3.0",
-    "ember-engines": "0.2.4",
-    "ember-export-application-global": "^1.0.4",
+    "ember-engines": "~0.3.4",
```

Comment:
The project was giving me errors with 0.2.x. I did update it and now it looks like it's not able to submit new item. Everything is working except creating a new item.